pybikes/fsm.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017, aronsky <aronsky@gmail.com>
+
+from lxml import etree
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class FSMSystem(BikeShareSystem):
+    sync = True
+    unifeed = True
+
+    meta = {
+        'system': 'fsm',
+        'company': ['FSM Ground Services Ltd.']
+    }
+
+    def __init__(self, tag, meta, feed_url):
+        super(FSMSystem, self).__init__(tag, meta)
+
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        scraper = scraper or utils.PyBikesScraper()
+
+        stations = []
+
+        data = scraper.request(self.feed_url)
+        dom = etree.fromstring(data.encode('utf-8'))
+        stations = self.get_stations(dom)
+        self.stations = list(stations)
+
+    def get_stations(self, dom):
+        ns = {'kml': 'http://www.opengis.net/kml/2.2'}
+        for placemark in dom.xpath('//kml:Placemark', namespaces=ns):
+            name = placemark.findtext('kml:name', namespaces=ns)
+            info = placemark.findtext('kml:description', namespaces=ns)
+            station_uid, bikes, free = utils.re.findall(r'\w+\:\s*(\d+)', info)
+            latitude, longitude = placemark.findtext( \
+                                  'kml:Point/kml:coordinates', \
+                                    namespaces=ns).split(',')
```

Comment:
Not really important, but since there's the issue with `bikes` and `free`, this style can be fixed too. No need to use backslashes here for line continuation:

```
latitude, longitude = placemark.findtext(
    'kml:Point/kml:coordinates',
    namespaces=ns
).split(',')
```
or even
```
latitude, longitude = placemark.findtext("""
    kml:Point/kml:coordinates
""", namespaces=ns).split(',')
```